src/components/form/businessForm.tsx

```
@@ -78,73 +78,100 @@ export function BusinessForm({ user_id }: { user_id: string }) {
     mode: 'all',
   });
 
+  // Fetch initial data
   useEffect(() => {
     const fetchData = async () => {
+      if (!user_id) return;
+      
```

ESLint

[error] Delete `······`

(prettier/prettier)